homeassistant/components/media_player/onkyo.py

```diff
@@ -55,6 +56,16 @@
 
 TIMEOUT_MESSAGE = 'Timeout waiting for response.'
 
+ATTR_VIDEO_OUTPUT = 'video_output'
+ACCEPTED_VALUES = ['no', 'analog', 'yes', 'out',
```

Comment:
is this valid for all onkyos?

Reply:
Those are the accepted values according to the onkyo-eiscp library, not all of them would work on all recievers i dont think (for me "out" switches to main and "out-sub" switches to sub, "sub" switches to both. so its not totally clear what to use but rather trial and error, planning to update this in the docs when i get time).
The command would only work for recievers with more than one hdmi-out port ofc.
Other than that i wouldn't really know as i only got one reciever to test with. (onkyo-eiscp said the command is for Japanese models but i got a European one and it works fine).